scripts/set-app-template-versions.js

```diff
@@ -70,14 +90,14 @@ const setVersion = (template, rootTmpDir) => {
     })
     .then(result => {
       if (result === 'skip') {
-        console.log(`${template} is already set to ${newVersion}, skipping`);
+        console.log(`${template} is already set to ${newVersions.nodeVersion}, ${newVersions.npmVersion}, and ${newVersions.coreVersion} for node, npm, and zapier-platform-core respectively, skipping`);
```

Comment:
it'd be nice to see `${newVersions.nodeVersion}, ${newVersions.npmVersion}, and ${newVersions.coreVersion} for node, npm, and zapier-platform-core` in some sort of function since its used a lot. There's also some variation between the messages, so it may not be worth it.